addons/mail/models/mail_thread.py

```diff
@@ -547,21 +547,21 @@ def _message_track_post_template(self, changes):
     def _track_template(self, changes):
         return dict()
 
-    def message_track(self, tracked_fields, initial_values):
+    def _message_track(self, field_names, initial_values):
```

Comment:
Why not "_track_message" then? Confusing? "_track_values"? "_track_all_the_things"?

Reply:
This is a "main API method". All main methods are message_something.